docs/quickstart.mdx

```diff
@@ -23,246 +23,341 @@ We will clone a preconfigured Node.js project with TypeScript support to get sta
 
-All packages you need for this guide are all configured and installed for you, check out the `package.json` file if you want to explore the dependencies.
+After running these two commands you should see the printed statement `Hello World!` in your terminal.
+Any changes made to the `index.ts` file in the src directory should now automatically run in your terminal upon saving.
+
+All packages you need for this guide are configured and installed for you. Check out the `package.json` file if you want to explore the dependencies.
 
 <details>
   <summary> Click to learn more about the packages </summary>
 
-- The account package will help you with creating smart contract accounts and an interface with them to create transactions.
-- The bundler package helps you with interacting with our bundler or alternatively another bundler of your choice.
-- The paymaster package works similarly to the bundler package in that you can use our paymaster or any other one of your choice.
-- The core types package will give us Enums for the proper ChainId we may want to use.
-- The modules package gives us access to the different modules we publish for the biconomy sdk.
-- The common package is needed by our accounts package as another dependency.
-- Finally the ethers package at version 5.7.2 will help us with giving our accounts an owner which will be our own EOA.
+- The **Account package** will help you with creating **Smart Account** and an interface with them to create transactions.
+- The **Bundler package** allows you to interact with our bundler or any other bundler of your choice.
+- The **Paymaster package** can be used with our paymaster or any other of your choice, similar to the bundler package.
+- The **Core Types package** will give us Enums for the proper ChainId we may want to use.
+- The **Modules package** provides access to the different modules that are published for the biconomy SDK.
+- The **Common package** is needed by our accounts package as another dependency.
+- Finally, with `version 5.7.2` of the `ethers` package, we can set our EOA as the account owner.
 
 </details>
 
-Let’s first set up a .env file in the root of our project, this will need a Private Key of any Externally Owned Account (EOA) you would like to serve as the owner of the smart account we create. This is a private key you can get from wallets like MetaMask, TrustWallet, Coinbase Wallet etc. All of these wallets will have tutorials on how to export the Private key. 
+Let’s first set up a `.env` file in the root of our project, this will need a **Private Key** of any **Externally Owned Account (EOA)** you would like to serve as the owner of the Smart Account we create. This is a private key you can get from wallets like MetaMask, TrustWallet, Coinbase Wallet etc. Each of these wallets will include step-by-step guides on how to export the Private key.
```

Comment:
```suggestion
Let’s first set up a `.env` file in the root of our project, this will need a **Private Key** of any **Externally Owned Account (EOA)** you would like to serve as the owner of the Smart Account we create. This is a private key you can get from wallets like MetaMask, TrustWallet, Coinbase Wallet, etc. Each of these wallets will include step-by-step guides on how to export the Private key.
```